### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by langningchen**

### CODE REVIEW

**Comment by langningchen** [comment]
Who will review?

File: .github/workflows/Translate.yml
Comment (boomzero) [inline .github/workflows/Translate.yml:1]: ...

File: index.html [hunk 1]
```diff
@@ -268,7 +268,7 @@ <h2 class="accordion-header">
             <h4>安装小明的OJ增强脚本</h4>
             <p>
                 安装用户脚本管理器后，您可以点击
-                <a href="https://web.xmoj-bbs.tech/小明的OJ.user.js" target="_blank">这里</a>
+                <a href="https://web.xmoj-bbs.tech/XMOJ.user.js" target="_blank">这里</a>
```
Comment (boomzero) [inline index.html:271]: I wonder how this happened(

**Review by boomzero — APPROVED** [review]

**Comment by boomzero** [comment]
好快啊

**Comment by langningchen** [comment]
因为我想把自动翻译去掉，所以先merge

**Pull request merged**